nautobot/extras/datasources/git.py

```diff
@@ -738,8 +624,8 @@ def update_git_config_context_schemas(repository_record, job_result):
 
 
 def import_config_context_schema(
-    context_schema_data, repository_record, job_result, logger  # pylint: disable=redefined-outer-name
-):
+    context_schema_data, repository_record, job_result
+):  # pylint: disable=redefined-outer-name
```

Comment:
is this disable still needed after removing the `logger` param?

Reply:
I don't fully understand why `import_local_config_context()` still has a `logger` param, while `import_config_context()` and `import_config_context_schema()` do not and instead now use the global `logger`? What's the pattern here that I'm missing?

Reply:
That was an oversight on my part. The only logger args that need to stay are for the `ensure_git_repository` and `git_repository_dry_run` methods that are both called by jobs.